src/sfctl/custom_app_type.py

```diff
@@ -68,7 +68,7 @@ def provision_application_type(client, #pylint: disable=too-many-locals,invalid-
             no_wait,
             application_type_build_path=application_type_build_path)
 
-    api_version = "6.1"
+    api_version = "6.2"
```

Comment:
Why are these changes in the remove SDK PR? Can you make these in a separate PR?